src/main/java/picard/fingerprint/CheckFingerprint.java

```diff
@@ -366,16 +368,15 @@ protected int doWork() {
 
             summaryFile.addMetric(metrics);
             log.info("Read Group: " + metrics.READ_GROUP + " / " + observedSampleAlias + " vs. " + metrics.SAMPLE + ": LOD = " + metrics.LOD_EXPECTED_SAMPLE);
-            if (metrics.LOD_EXPECTED_SAMPLE!=0) {
-                anyNonzeroLod |= true;
+            if (metrics.LOD_EXPECTED_SAMPLE != 0) {
```

Comment:
you could write
```
anyNonzeroLod |= metrics.LOD_EXPECTED_SAMPLE != 0;
```
(also, I think the proper spelling is `anyNonZeroLod` as non-zero is a hyphenated word)